### ISSUE TITLE
PC Freeze

### ISSUE THREAD

**Issue by stephdotnet** [Jays2Kings/DS4Windows#150]
Hi,

I'm on windows 7 and i'am having sever issues when launching the app (instant freeze i have to hard reboot my pc), or on connecting the device (DS4).

Where can i find the error log so i can send it to you?

**Comment by empz** [comment]
I just discovered DS4Windows was the cause that my whole PC freezes when launching certain games having DS4Windows running in exclusive mode (Hide DS4 controller). 
My guess it's that during some games launching process, it wants to gain exclusive control of the joysticks and enter in some kind of "fight limbo" with DS4Windows and the whole thing freezes the entire computer.

As I said this only happens when the "Hide DS4 controllers" is checked. 

The games in which this issue happen already supports the DS4 natively, so stopping/closing DS4Windows is my current workaround. But again it's not ideal since I have to manually start/stop depending on which game I'm about to launch.

**Comment by ghost4luck** [comment]
Same issue. If DS4Windows running in exclusive mode and I launch Uplay my computer freeze for some time. But if DS4Windows running without "Hide DS4 Controller" option it works well, Uplay starts without freezing.

**Comment by Dalai-** [comment]
Can confirm, with DS4Windows running, UPlay and recent Ubisoft games freeze up on launch. Bizarrely, browsing to a Kickstarter campaign page also causes a ~15 second freeze. If I close down DS4Windows these freezes don't happen. This means I can't really leave DS4Windows running all the time.

**Comment by rbross3030** [comment]
Just wanted to confirm this as well. I am running Windows 7 x64 SP1 and when I open UPlay with DS4 running in "Hide DS4 Controller" mode my computer freezes for about 15 seconds. It also locks up any time I power on a controller after it turns off from power saver. I'd really prefer to use DS4Windows over using native PS4 support because it seems that the Xbox controller support is much more universal than PS4 and works on more games.

**Comment by Rose2234** [comment]
yeah this project is DEAD and he does not care anymore i rather give my money to a developer that ACTUALLY cares about his work and not give out cliche excuses like "oh i have been way too busy". and his site does not work anymore it says Bandwidth has been exceeded the limit. Xpadder here i come!!! AND THAT INSTALL ISSUE IS SILL IN EFFECT

**Comment by rbimas37** [comment]
http://puu.sh/w5XkX/982d37614a.png

Solved the problem, only need to have these windows updates (minus the language pack).
Helped for me!

**Comment by WVVWVWVVW** [comment]
Got all of those windows updates installed, but using exclusive mode still makes my computer freeze when starting uplay.

**Comment by Lightbass** [comment]
Hello. Sorry for bad english.
I have same problem too, i tryed many programs and SCP Drivers may help me, but for me, it didn't worked.
Then i found this solution for Input mapper:
1) Download and install Input mapper 1.6.10
2) Download and install HidGuardian 1.0.6230.17641
3) And if you have DS4 v2, try this:
You have to edit the AffectedDevices registry value via regedit in order for HidGuardian to take control of the device.
HKEY_LOCAL_MACHINESYSTEMCurrentControlSetServicesHidGuardianParametersAffectedDevices
The file generated by InputMapper does not include the device instance path for the version 2 DS4 controller so HidGuardian will not be used. Here are the lines that should be used for a version 2 controller. The first 3 lines is for a USB connection and the next line is for a BT connection.

HIDVID_054C&PID_09CC&REV_0100&MI_03
HIDVID_054C&PID_09CC&MI_03
HIDVID_054C&UP0001_U0005
HID{00001124-0000-1000-8000-00805F9B34FB}_VID&0002054C_PID&09CC 

4) Reboot.

This may work unstable, but this only one fix that i found after many hours on google and this worked for me. Links:
https://inputmapper.com/downloads/inputmapper-1-5/download/2-input-mapper/73-inputmapper
https://inputmapper.com/downloads/download/5-tools/76-inputmapper-hidguardian
Fullpack: https://drive.google.com/open?id=16K2Yh-Yn936qdrCpdGOtc_lGLq8ZERoI
